src/test/rpc/Tx_test.cpp

```diff
@@ -528,6 +547,29 @@ class Tx_test : public beast::unit_test::suite
                 client.status.ok(), client.reply);
         };
 
+        auto grpcAccountTx = [&grpcPort](
```

Comment:
This lambda doesn't get used until much further down in the file.  Maybe it should be moved down closer to where it is used, around line 693.